benchmarks/lasso/solvers/baseline.py

```diff
@@ -15,14 +21,21 @@ def set_objective(self, X, y, lmbd):
     def run(self, n_iter):
         n_features = self.X.shape[1]
         z_hat = np.zeros(n_features)
-        for i in range(n_iter):
+        t_new = 1
+        for _ in range(n_iter):
+            if self.use_acceleration:
+                t_old = t_new
+                t_new = (1 + np.sqrt(1 + 4 * t_old ** 2)) / 2
+                z_old = z_hat.copy()
             grad = self.X.T.dot(self.X.dot(z_hat) - self.y)
             z_hat -= 1 / self.L * grad
             z_hat = self.st(z_hat, 1 / self.L * self.lmbd)
+            if self.use_acceleration:
+                z_hat = z_hat + (t_old - 1.) / t_new * (z_hat - z_old)
         self.z_hat = z_hat
 
     def st(self, z, mu):
-        return np.sign(z) * np.maximum(0, abs(z) - mu)
+        return np.sign(z) * np.maximum(0, np.abs(z) - mu)
```

Comment:
Just out of curiosity, does this change anything? I tend to use `abs` instead of the other but only out of habit and fear for longer lines ^^

Reply:
it's the same but it's more readable as it makes explicit that it works on array

Reply:
AFAIK there is no time difference